src/app/modules/filters/filters-popover/filters-popover.component.ts

```diff
@@ -6,33 +6,40 @@ import { Component } from '@angular/core';
   styleUrls: ['./filters-popover.component.scss']
 })
 export class FiltersPopoverComponent {
-  filtersVisible = false;
   filtersBoxVisible = false;
+  popupContainerWidth = '0em';
```

Comment:
Let's keep css stuff in css unless its truly needed. @axdanbol is there a better way to do this?

Reply:
On container add `[class.visible]="filtersBoxVisible"` then everything else should be possible to move to css.
I.e.
```scss
.popover-container {
  ...
  .popover-body {
     ...
  }

  &.visible {
    width: 35rem;

    .popover-body {
        opacity: 100%;
        ...
    }
  }
}

Reply:
good idea.  I made those changes in the latest push.